client/src/components/Schedule/FormEntity.tsx

```diff
@@ -377,20 +377,19 @@ const createTask = async (courseId: number, values: any, isUpdateMode: boolean,
     attributes: JSON.parse(values.attributes ?? '{}'),
   } as Partial<Task>;
 
-  let taskTemplateId;
+  let taskId = undefined;
 
   if (isUpdateMode && editableRecord) {
-    taskTemplateId = editableRecord.id;
-    await taskService.updateTask(taskTemplateId, templateTaskData);
+    await taskService.updateTask(editableRecord.id, templateTaskData);
   } else {
-    const data: any = await taskService.createTask(templateTaskData);
-    taskTemplateId = data.identifiers[0].id;
+    const { id } = await taskService.createTask(templateTaskData);
```

Comment:
On my local app, I resive InsertResult with identifiers, generatedMaps, raw. And nothing works with  { id } = await changes... I suggest restore previous variant.